include/wx/stdpaths.h

```diff
@@ -257,7 +257,10 @@ class WXDLLIMPEXP_BASE wxStandardPaths : public wxStandardPathsBase
     {
         return m_prefix + wxS("/") + basename;
     }
-
+    virtual wxString GetSharedLibrariesDir() const override { return m_prefix; }
+#if defined(__WINDOWS__)
+    static wxString MSWGetShellDir(int WXUNUSED(csidl)) { return wxEmptyString; }
+#endif
```

Comment:
What's this and why do we need it? It doesn't seem to be used anywhere.

Reply:
@vadz,
When you turn off wxUSE_STDPATH and compile on Windows this function has to be defined.
Otherwise you will get compiler error

Reply:
Ah, so this has nothing to do with this PR.

This is not the proper fix for the problem, of course. AFAICS this is only used in `src/msw/rt/notifmsgrt.cpp` and so the right thing to do would be to add `#if wxUSE_STDPATH` around the code there. This really has nothing to do here in any case.

Reply:
@vadz ,
No it doesn't..
However, when I set wxUSE_STDPATH to 0 and tried to compile with MSVC I got an error.

All I was trying to do is make sure the code compiles. And when I saw it didn't - I tried to fix it.

I can remove this piece of code here and add the ifdef in there in another PR fixing it. However, I will not be able to test it